src/vs/workbench/contrib/notebook/browser/view/renderers/backLayerWebView.ts

```diff
@@ -215,7 +215,7 @@ export class BackLayerWebView<T extends ICommonCellInfo> extends Themable {
 	}
 
 	private _logRendererDebugMessage(msg: string) {
-		this._renderLogger.debug(`${this.id} - ${msg}`);
+		this._renderLogger.debug(`${this.documentUri} (${this.id}) - ${msg}`);
```

Comment:
`this.documentUri.toString()`? Won't this output `[object...]`?

Reply:
JS implicitly calls `.toString()` on the uri in this case. Here's an example log: 

```
2022-11-28 15:49:28.329 [debug] file:///Users/matb/projects/san/abc.ipynb (8255fd85-47a9-47ef-84f2-e075a425a84e) - [renderer vscode.markdown-it-cell-attachment-renderer] - Imported renderer {
    "duration": "19.600000023841858ms"
}
```